test/k8sT/Policies.go

```diff
@@ -591,6 +605,82 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 			testConnectivitytoRedis()
 		})
 	})
+
+	Context("KubernetesNetworkPolicy between server and client", func() {
+		BeforeEach(func() {
+			kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
```

Comment:
This is already done in initialize function

Reply:
Done